lib/install.js

```diff
@@ -242,8 +242,15 @@ Installer.prototype.run = function (_cb) {
     }
   }
 
+  var preinstallSteps = []
   var installSteps = []
   var postInstallSteps = []
+
+  preinstallSteps.push(
```

Comment:
Actually let me edit that... it does make some sense because, in the case where you want to bail before install, then I think having it organized this way will allow for this whereas having everything chained in a single install step would mean you'd wait longer for that feedback?